t/t2717-python-cli-plugins.t

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+test_description='Test command line plugin interface'
+
+. $(dirname $0)/sharness.sh
+
+export FLUX_CLI_PLUGINPATH=${SHARNESS_TEST_SRCDIR}/cli-plugins/
+
+test_under_flux 4 job
+
+test_expect_success 'flux-alloc: base --help message is formatted correctly' '
+  flux alloc --help 
+'
```

Comment:
This should check something ;-) Maybe that the plugin option section appears in the output.